DICOM/src/Microsoft.Health.DeID.SharedLib/RedactFunction.cs

```diff
@@ -0,0 +1,147 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+using EnsureThat;
+using Microsoft.Health.Dicom.DeID.SharedLib.Models;
+using Microsoft.Health.Dicom.DeID.SharedLib.Settings;
+
+namespace Microsoft.Health.Dicom.DeID.SharedLib
+{
+    public class RedactFunction
+    {
+        private static readonly int AgeThreshold = 89;
+        private static readonly string ReplacementDigit = "0";
```

Comment:
🟡 [Suggestion]
Better to make `ReplacementDigitand` and `InitialDigitsCount` **const** instead of **readonly** unless we have clear reason.